bibnumber/batch.cpp

```diff
@@ -243,10 +245,13 @@ int process(std::string inputName, std::string svmModel) {
 				bimaps::multiset_of<int> > imgTagBimap;
 
 		imgTagBimap tags;
+        bool file_dump_flag;
```

Comment:
maybe you can fix the indentation here?

Reply:
Hi @gheinrich,
I have fixed the indentation issues and pushed the  changes.

Thanks,
Rohit